### PR TITLE
ROX-25275: Add permissions to read ACS addon objects for ACS team

### PR DESCRIPTION
**Pull request by kurlov**
### What type of PR is this?
_feature_

### What this PR does / why we need it?
ACS team should be able to list and view ACS Addon related resources

### Which Jira/Github issue(s) this PR fixes?

[ROX-25275](https://issues.redhat.com/browse/ROX-25275)

### Pre-checks (if applicable):
~- [ ] Tested latest changes against a cluster~
- [ ] Included documentation changes with PR
~- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:~

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

File: deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml [hunk 1]
```diff
@@ -45,6 +45,26 @@ rules:
   - get
   - list
   - watch
+# ACS Team can view ACS Addon CRs
+- apiGroups:
+  - package-operator.run
+  - addons.managed.openshift.io
```
Comment (erdii) [inline deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml:51]: i think the rules should be separated by apigroup so it's clear which resources belong to the `package-operator.run` group and which ones belong to `addons.managed.openshift.io`

**Review by erdii — COMMENTED** [review]
I don't know which clusters in our managed fleet receive these permissions via ACM so I don't know how relevant the following is:

The cluster selector for the ACM PlacementRule [1] only contains the prod version of the addon.

[1] https://github.com/openshift/managed-cluster-config/blob/a5db7d40837d82bd4bab81be10edb7d274de2dec/deploy/backplane/acs/config.yaml#L11

File: deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml (hunk 1 above)
Comment (kurlov) [reply deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml]: Splitted it into two groups

**Pull request merged**